src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,73 @@
-import { css, Theme } from '@emotion/react';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    opacity: 1,
+  },
+  '50%': {
+    opacity: 0,
+  },
+  '100%': {
+    opacity: 1,
+  },
+});
 
 const styles = {
-  liveLabel: ({ palette, spacings, fontVariants }: Theme) =>
+  liveLabelText: ({ palette, spacings, fontVariants }: Theme) =>
     css({
       color: palette.LIVE_DARK,
       display: 'inline-block',
       marginInlineEnd: `${spacings.FULL}rem`,
       ...fontVariants.sansBold,
     }),
+  liveLabelCircle: ({ palette, spacings }: Theme) =>
+    css({
+      color: palette.LIVE_DARK,
+      borderRadius: '50%',
+      display: 'inline-block',
+      width: '16px',
+      height: '16px',
+      background: `radial-gradient(circle,transparent 53%,hsla(180, 100%, 20%,.5) 53.5%, ${palette.LIVE_DARK} 54%)`,
+      marginInlineEnd: `${spacings.HALF}rem`,
+      position: 'relative',
+    }),
+
+  firstPromo: ({ mq }: Theme) =>
+    css({
+      [mq.GROUP_1_ONLY]: {
+        width: '18px',
+        height: '18px',
+      },
+      [mq.GROUP_2_ONLY]: {
+        width: '24px',
+        height: '24px',
+      },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        width: '28px',
+        height: '28px',
+      },
+    }),
+
+  livelabelPulse: ({ palette }: Theme) =>
+    css({
+      '&:before': {
+        content: `''`,
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        borderRadius: '50%',
+        background: `radial-gradient(circle,${palette.LIVE_DARK} 37%,rgba(0,102,102,.5) 37.5%,transparent 38%)`,
```

Comment:
As above, can we remove the `rgba(0,102,102,.5)` and just use `palette.LIVE_DARK` instead?